torch/csrc/autograd/functions/batch_normalization.cpp

```diff
@@ -186,12 +195,22 @@ auto BatchNormBackward::apply(const variable_list& grad_outputs) -> variable_lis
         eps);
   }
 
+  // Add saved variables used out of the pure autograd to inputs
+  variable_list all_inputs(grad_outputs);
+  all_inputs.push_back(input_var);
+  if (weight.get()) {
+    all_inputs.push_back(weight_var);
+    all_inputs.push_back(bias_var);
```

Comment:
Biases are actually not used in this function.
Shouldn't they be removed from here and from the double backward function?